test/with_dummyserver/test_poolmanager.py

```diff
@@ -22,6 +22,27 @@ def setup_class(cls):
         cls.base_url = "http://%s:%d" % (cls.host, cls.port)
         cls.base_url_alt = "http://%s:%d" % (cls.host_alt, cls.port)
 
+    def test_redirect_preload(self):
+        def build_request(number, preload):
+            return http.request(
+                    "GET",
+                    "%s/redirect" % self.base_url,
+                    fields={"target": "%s/?page=%d" % (self.base_url, number)},
+                    preload_content=preload
+                )
+
+        with PoolManager(block=True, maxsize=3) as http:
+            for j in range(1, 5):
+                r1 = build_request(number=j, preload=True)
+                r2 = build_request(number=j, preload=False)
+
+                data = bytes("Dummy server! page %d" % j, "utf-8")
```

Comment:
Writing `b"Dummy server! page %d" % j` here instead should fix the Python 2.7 builds